README.md

```diff
@@ -1,2 +1,99 @@
 # Shape-Based-hand-Gesture-Recognition
 Srishti 2020
+
+## Abstract
+   
+   <p align="justify">The aim of the project is to create hand bot which can replicate the movements of hands of the person who is interacting with the bot, with the help of image processing. This project is a combination of live motion detection and gesture identification.</p>
+
+![](https://github.com/Rohit-Kumar-219/Shape-Based-hand-Gesture-Recognition/blob/master/Images/Image%20identification.jpg)
```

Comment:
Your image URL is wrong! Update it with new URL

Reply:
Update with ``![](/Images%20and%20Videos/Images/Image%20identification.jpg)`` to make it working